ui/components/Navigator.js

```diff
@@ -1168,6 +1169,7 @@ class Navigator extends React.Component {
     )
     return (
       <NoSsr>
+        { (!("open" in other) || other.open) &&
```

Comment:
What is other here?

Reply:
other is rest of the props passed into Navigator component. It's defined above

Reply:
oh okay, thanks :+1:

Reply:
`!("open" in other)` does this affects the functionality? 
I think the two sides of `OR` condition are somewhat same, or is it required.

Reply:
1. The first condition checks if the key (i.e. "open") is not present (The key is not present in desktop mode, so it will always show the chevron)

2. The second condition checks if the key is present then is it also true (The key is present which means currently mobile mode is visible, then it checks if the drawer is open `other.open` ,  so it shows the chevron) 

It tested it both with and without the the first condition and found that it is required for correct functioning